src/components/WebsiteTabs/WebsiteTabs.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Tabs } from 'carbon-components-react';
+
+/**
+ * remove wrapper elements from internal content:
+ * - ibm-row // row-wrapper divs generated in Markdown.js
+ * - ibm-row > ibm--col* // column dives that are direct children to rows, which would be generated & placed in Markdown.js
+ * */
+
+export default class WebsiteTabs extends React.Component {
+  render() {
+    let tabChildren = [];
+
+    this.props.children.forEach(child => {
+      if (child.type && child.type.displayName == 'Tab') {
```

Comment:
Just wanted to see if you are interested in [`Array.prototype.filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter).